File: src/main/java/org/goafabric/core/data/controller/vo/Address.java
package org.goafabric.core.data.controller.vo;

import jakarta.validation.constraints.NotNull;

public record Address (
        String id,

        @NotNull String use,
        @NotNull String street,
        @NotNull String city,
        @NotNull String postalCode,
        @NotNull String state,
        @NotNull String country
) {}

File: src/main/kotlin/org/goafabric/core/organization/controller/dto/Address.kt
package org.goafabric.core.organization.controller.dto

data class Address(
    var id: String? = null,
    var version: Long? = null,
    var use: String? = null,
    var street: String? = null,
    var city: String? = null,
    var postalCode: String? = null,
    var state: String? = null,
    var country: String? = null
)
